x-pack/plugins/index_management/public/application/components/index_templates/simulate_template/simulate_template.tsx

```diff
@@ -37,11 +36,6 @@ export const SimulateTemplate = React.memo(({ template, filters }: Props) => {
     }
 
     const indexTemplate = serializeTemplate(stripEmptyFields(template) as TemplateDeserialized);
-
-    // Until ES fixes a bug on their side we will send a random index pattern to the simulate API.
-    // Issue: https://github.com/elastic/elasticsearch/issues/59152
-    indexTemplate.index_patterns = [uuid.v4()];
```

Comment:
This issue is still open, but I moved this logic to the server because that way the API route itself is responsible for preventing bad input from being sent to ES.